controller/tests/utils.py

```diff
@@ -28,6 +33,29 @@ def get_mock_mediator_response_snapshot(capacity, name, wwn, volume_name, array_
     return snapshot
 
 
+def get_mock_array_connection_info(user="user", password="pass", array_addresses=None, system_id="u1"):
+    if array_addresses is None:
+        array_addresses = ["arr1"]
+    return ArrayConnectionInfo(array_addresses=array_addresses, user=user, password=password, system_id=system_id)
```

Comment:
is it really a mock?

Reply:
in the sense that its not the real return value from utils

Reply:
then rename `mock` to something like `fake`/`test`

Reply:
done